setup.py

```diff
@@ -1,59 +1,3 @@
-"""
-Build and install the project.
-"""
-from setuptools import find_packages, setup
+from setuptools import setup
 
-NAME = "pygmt"
-FULLNAME = "PyGMT"
-AUTHOR = "The PyGMT Developers"
-AUTHOR_EMAIL = "leouieda@gmail.com"
-MAINTAINER = AUTHOR
-MAINTAINER_EMAIL = AUTHOR_EMAIL
-LICENSE = "BSD License"
-URL = "https://github.com/GenericMappingTools/pygmt"
-DESCRIPTION = "A Python interface for the Generic Mapping Tools"
-KEYWORDS = ""
-with open("README.rst", "r", encoding="utf8") as f:
-    LONG_DESCRIPTION = "".join(f.readlines())
-
-PACKAGES = find_packages(exclude=["doc"])
-SCRIPTS = []
-PACKAGE_DATA = {"pygmt.tests": ["data/*", "baseline/*"]}
```

Comment:
Is it possible to put this `PACKAGE_DATA` line in `pyproject.toml`?

Reply:
Done in f257a361b2ef870b0f9d159835fd5cbac065c5cb.